src/components/LessonDetails/LessonDetails.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from "react";
+
+import Grid, { GridArea } from "../Grid";
+import Flex from "../Flex";
+import Heading from "../Typography";
+import KeyLearningPoints from "../KeyLearningPoints/KeyLearningPoints";
+import CommonMisconceptions from "../CommonMisconceptions/CommonMisconceptions";
+import KeyWords from "../KeyWords/KeyWords";
+
+export type KeyLearningPoint = {
+  keyLearningPoint: string | null;
+};
+
+export type CommonMisconception = {
```

Comment:
Should these live with their respective components ? Better to have a one way dependency which follows the component hierarchy

Reply:
Agreed

Reply:
Moved them into respective components ✅